Woof/Woof/ViewModel/DetailPetSitterViewModel.swift

```diff
@@ -8,8 +8,8 @@ final class DetailSitterViewModel: ObservableObject {
     /// The additional information about the sitter.
     @Published var bio: String
 
-    /// The rating of the sitter, on a scale from 0 to 5.
-    @Published var rating: Int
+    /// The rating of the sitter, on a scale from 1 to 5 star, or that the rating is not available.
```

Comment:
```suggestion
    /// A 5-Star rating of the pet sitter.
```

All the details I can see in the `StarRating`'s documentation , you don't have to repeat them.

Reply:
Fixed.